shared/actions/chat2/index.js

```diff
@@ -2106,6 +2107,39 @@ function* handleSeeingExplodingMessages(action: Chat2Gen.HandleSeeingExplodingMe
   })
 }
 
+const loadStaticConfig = (action: ConfigGen.BootstrapPayload, state: TypedState) => {
+  if (state.chat2.staticConfig) {
+    // don't need it
+    return null
+  }
+  return Saga.call(RPCChatTypes.localGetStaticConfigRpcPromise)
+}
+
+const loadStaticConfigSuccess = (res: ?RPCChatTypes.StaticConfig) => {
+  if (!res) {
+    // we already had it
+    return
+  }
+  if (!res.deletableByDeleteHistory) {
+    logger.error('chat.loadStaticConfig: got no deletableByDeleteHistory in static config')
```

Comment:
this would have the affect of deleting all message types on getting a delete history until bootstrap happens again. but it should never happen.